src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -357,20 +361,27 @@ private void UpdateLayoutModels()
             }
         }
 
-        private void ParseDeviceInfoData()
+        private void ParseDeviceInfoData(bool debug = false)

```

Comment:
Let's replace the `bool` with an enumeration `ParseDeviceMode`.

Reply:
Done